Sources/Networking/Core/RequestURLParametersType.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  RequestURLParametersType.swift
+//
+//
+//  Created by Matej Molnár on 11.12.2023.
+//
+
+import Foundation
+
+// MARK: - Defines various parameter types to be sent in the URL
```

Comment:
Why are we using MARKs on top of the file?
My understanding of MARKs is to use them to split the file into logical parts but I noticed that we use it like this (eg. top of the file) in other places as well.

Is there a reason for that that I'm missing?

Reply:
Agreed, I don't see good reason for the MARKs either. Shall I create another PR to remove these unnecessary MARKs?

Reply:
Do you know who started this convention? 😃 As I don't want to overrule that without first asking, but otherwise I would be for this in a new PR 😊.

Reply:
Not sure, maybe @cejanen?